sfge/src/main/java/com/salesforce/graph/ops/MethodUtil.java

```diff
@@ -531,6 +527,16 @@ public static Optional<ApexValue<?>> getApexValue(
             apexValue = symbols.getReturnedValue((InvocableVertex) vertex).orElse(null);
         }
 
+        if (apexValue == null && vertex instanceof MethodCallExpressionVertex) {
```

Comment:
This portion makes sure that a method call on a class instance array item gets a corresponding method definition. (See javadoc on test for full explanation)